forge-game/src/main/java/forge/game/ability/effects/CopySpellAbilityEffect.java

```diff
@@ -72,7 +72,11 @@ public void resolve(SpellAbility sa) {
 
         List<SpellAbility> tgtSpells = getTargetSpells(sa);
 
-        tgtSpells.removeIf(tgtSA -> StaticAbilityCantBeCopied.cantBeCopied(tgtSA.getHostCard()));
```

Comment:
wouldn't it be better to get removeIf working for us?

Reply:
good point